src/column.ts

```diff
@@ -120,21 +120,27 @@ export class ColumnFactory {
 
 export class RawColumn {
     public cellLength: number = 0;
-    constructor(public columnValues: string[]) { 
+    public cellLengths: number[] = [];
+    constructor(public columnValues: string[]) {
         this._setCellLength();
     }
 
     isEmpty(): boolean {
         return this.cellLength == 0;
     }
 
+    getLenth(columnValue: string): number {
+        return columnValue.replace(/[^\x00-\xff]/g, "aa").length; // 双字节长度为2

```

Comment:
Replacing it with "aa" is just a hack. It should be tested separately to return a given length.